Tasks/PackerBuildV0/package.json

```diff
@@ -5,7 +5,7 @@
     "@types/node": "^10.17.0",
     "@types/mocha": "^5.2.7",
     "@types/q": "^1.0.7",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.0-preview.0",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "1.178.1",
```

Comment:
I was under impression that the change is to start using npm packages instead of common file modules. But I was wrong. Does it mean that 2.0.0-preview.0 version of arm-rest-v2 package has not got Anurag's fix? Should we not fix that instead of the task here?
@anuragc617 @ddivanshu